src/components/organisms/SigninForm/index.tsx

```
@@ -0,0 +1,50 @@
+
+  const onSubmit = (data: SigninFormData) => {
+    const { username, password } = data;
+    // onSigninが存在する場合のみ実行
+    onSignin && onSignin(username, password);
+  }
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <input
+        type='text'
+        {...register('username', { required: 'ユーザー名は必須です' })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)